{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "24e58390_bf1d5c86",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1001591
      },
      "writtenOn": "2021-10-19T23:21:59Z",
      "side": 1,
      "message": "PTAL once more! This one is a tiny bit more involved than some of the previous error updates, but not by much.",
      "revId": "6be2231f0765ba34fc0a3123e0b738713b398895",
      "serverId": "dd02978d-1a8e-36d7-bcc0-a5723e5c0abd"
    }
  ]
}